assembly/src/assembler/instruction/u32_ops.rs

```diff
@@ -518,8 +384,39 @@ fn handle_division(
         _ => unreachable!("unsupported operation mode"),
     }
 
-    span.push_op(U32div);
+    span.add_op(U32div)
+}
+
+// BITWISE OPERATIONS - HELPERS
+// ================================================================================================
 
+/// Prepare a bitwise operation argument, extracting it either from the stack, or from the provided
+/// immediate value.
+///
+/// This function supports only checked and unchecked modes; if some other mode is provided, it
+/// will panic.
```

Comment:
Would be great to add a bit more detail about what the function does. For example, I think it computes `2^b` and leaves it on the top of the stack, right? (where `b` can be either an immediate value or the value at the top of the stack).